prime-router/src/testIntegration/kotlin/FileNameTemplateIntegrationTests.kt

```diff
@@ -349,6 +359,80 @@ class FileNameTemplateIntegrationTests {
         assertThat(fileName).startsWith("cdcprime_laoph_production_$formattedDate")
     }
 
+    @Test
+    fun `test standard name format when schemaBaseName is present`() {
+        // arrange
+        val key = "standard"
+        val config = mockkClass(Hl7Configuration::class).also {
+            every { it.schemaName }.returns("classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml")
+        }
+        assertThat(metadata.fileNameTemplates).containsKey(key)
+        val standardNameFormat = metadata.fileNameTemplates[key]
+
+        // act
+        val fileName = standardNameFormat?.getFileName(config, reportId = reportId)
+            ?: assertk.fail("error getting standard file name")
+
+        // assert
+        assertThat(fileName).startsWith("oru_r01-base.yml-$reportId-$formattedDate")
```

Comment:
realize this is out of scope for the ticket. But it seems odd to include the ext for the schema file in the filename. Maybe we need a ticket to investigate that?

Reply:
Agreed. I will log this tomorrow.

Reply:
Logged: https://app.zenhub.com/workspaces/platform-6182b02547c1130010f459db/issues/gh/cdcgov/prime-reportstream/14703